Tests/js/models/assets/ez-contentmodel-tests.js

```diff
@@ -820,15 +820,11 @@ YUI.add('ez-contentmodel-tests', function (Y) {
                 }
             };
 
-            this.query = {
-                'body': {
-                    ViewInput: {
-                        'LocationQuery': {
-                            'Criteria' : ""
-                        }
-                    }
-                }
-            };
+            this.query = new Y.Mock();
+            Mock.expect(this.query, {
+                method: 'setFilter',
+                args: [Mock.Value.Object],
```

Comment:
this is too weak, you should also check that the passed object contains the key `ContentIdCriterion` with the content id as its value.